### ISSUE TITLE
Putty config not working

### ISSUE THREAD

**Issue by luku** [SchoofsKelvin/vscode-sshfs#71]
OS: Windows 10 (64-bit)
VS Code 1.28.2 (user setup)
Putty (64-bit) 0.70

I have following config in VS Code:
```json
"sshfs.configs": [
        {
            "name": "putty",
            "putty": "My putty session"
        },
        {
            "name": "manual",
            "label": "Manual config session",
            "host": "my.host.org",
            "username": "my.user",
            "privateKeyPath": "/root/.ssh/myPrivateKey",
            "root": "~/"
        }
    ]
```
When trying to connect via *manual* configuration all works fine. But when connecting via *putty* configuration (which has exactly the same values for host, username, private key) I get following error: Error while connecting to SSH FS putty: getaddrinfo EAI_FAIL my.user@my.host.org:22 (EAI_FAIL seems to be *A nonrecoverable failure in name resolution occurred.*)
All works fine when opening session via Putty directly. I use no proxy and all other Putty configuration settings are default.
I tried to supply IP instead of domain name, but same error occurs. Same when I run VS Code as administrator.
Any ideas what could be problem? Are there any other ports except 22 that needs to be open in firewall when using this extension in VS Code?

**EDIT:** I tried to reproduce this issue on another computer with same setup, but different host and it's working fine. So my problem must be related to DNS resolving or firewall. I will let you know when I know more.

**Comment by SchoofsKelvin** [comment]
Are you using `username@host` as host in the PuTTY session? That's what the error seems to indicate is going wrong here.

**Comment by luku** [comment]
You're right! It was due to `username@host` format in Putty session. But now, it prompts me to provide the username twice. 
In order to fix that I had to change Putty session configuration: Connection > Data > Auto-login username into *my.user*. Adding standard user inside `~/.ssh/config` doesn't help. It's also possible to add the username inside sshfs config:
```
"sshfs.configs": [
        {
            "name": "putty",
            "putty": "My putty session"
            "username": "my.user",
        }
]
```
but it's not as good as the Auto-login in Putty, since I want both Putty and VS Code to connect directly as *my.user*.

Also setting private key for authentication inside Putty > Connection > SSH > Auth doesn't work, the key must be always added via Pageant (which I didn't know).

**Comment by SchoofsKelvin** [comment]
A few things here:
1. The extension (currently) ignores any (extern) config files, like `~/.ssh/config`
2. If `Connection > Data > "Use system username"` is used, it should use `process.env.USERNAME`
3. Pageant has priority over private keys (for now?), it's difficult to work around that with the library I use

I just tested something for item 2, and it seems I need to use `process.env.USER` on unix systems. I've just committed a fix (037eeb5), included in version 1.11.2 of the extension, which is now live.
